frontend/pages/DashboardPage/cards/Software/_styles.scss

```diff
@@ -63,12 +64,24 @@
       }
 
       tbody {
-        .name__cell,
+        .name__cell {
+          // overflow: hidden;
```

Comment:
These and the version cell are commented to show the values that are being inherited?

Reply:
Oh whoops, meant to take those out, that code was used to not generate a tooltip and are being replaced with  TruncatedTooltipCell or whatever it's called that doesn't need it